.github/workflows/explorer-client-prs.yml

```diff
@@ -51,18 +44,23 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+      - uses: pnpm/action-setup@v2.2.2
+        with:
+          version: 7
       - name: Install Nodejs
         uses: actions/setup-node@v3
         with:
           node-version: "16"
-          cache: "yarn"
-          cache-dependency-path: ./explorer/client/yarn.lock
+          cache: "pnpm"
+      - name: Install dependencies
```

Comment:
should we also update the deployment script in the operations repo?

Reply:
Yep, I'll make a corresponding PR on that for that.